aries-site/src/examples/components/button/DosDonts/ButtonBadCancelPreview.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, Button, Heading, Header, Text } from 'grommet';
+
+export const ButtonBadCancelPreview = () => (
+  <Box width="large" gap="medium" pad="small">
+    <Header
+      direction="column"
+      align="start"
+      gap="xsmall"
+      pad={{ horizontal: 'xxsmall' }}
+    >
+      <Heading level={2} margin="none" size="small">
+        Delete item?
+      </Heading>
+      <Text size="large">
+        Deleting this item cannot be undone. Consider previewing results to see
```

Comment:
Now that we've removed the "Preview Results" button, let's remove the "Consider previewing ..." sentence.